### ISSUE TITLE
nuget package does not install to all frameworks

### ISSUE THREAD

**Issue by dominicaguilar** [marcojak/MauiMTAdmob#107]
In Visual Studio, when adding the NuGet package, it installs to Windows and Android frameworks, but it is not loaded to ios framework. This happen whether the package is loaded to the project or the solution.

After Nuget package is installed; under Dependancies/Packages  "Plugin.MauiMTAdmob (1.7.0)" only appears in the "net8.-0-android" and "net8.0-windows10.0.19041.0", but not under "net8.0-ios".

In the project file th is what is added:

	<ItemGroup Condition="'$(TargetFramework)' == 'net8.0-android'">
	  <PackageReference Include="Plugin.MauiMTAdmob">
	    <Version>1.7.0</Version>
	  </PackageReference>
	</ItemGroup>

	<ItemGroup Condition="'$(TargetFramework)' == 'net8.0-windows10.0.19041.0'">
	  <PackageReference Include="Plugin.MauiMTAdmob">
	    <Version>1.7.0</Version>
	  </PackageReference>
	</ItemGroup>

If I add it in manually the project will error on restore.

**Comment by marcojak** [comment]
You can add the package reference without specifying a condition...it just
works.

I've just checked and the supported frameworks are:

TargetFrameworks:
net8.0-android;net8.0-ios;net8.0-maccatalyst;net9.0-android;net9.0-ios;net9.0-maccatalyst

Try without the conditions.
If it doesn't work, I'll have a look tomorrow

**Comment by dominicaguilar** [comment]
I don't add the conditions, those are added by the Nuget manager. I simple to to the nuget manager find the package and install it.

**Comment by marcojak** [comment]
In you csproj, remove all that part and just add:

<PackageReference Include="Plugin.MauiMTAdmob">
    <Version>1.7.0</Version>
  </PackageReference>

It should automatically work for all frameworks. Not sure why for you it
created all those conditions.

**Comment by dominicaguilar** [comment]
When I did as you suggested, the true problem showed it's ugly face. The problem was that I was running .net8.0 version 17.5 and the package only supports .net8.0 version 18.0, so the restore failed. That is why it added the conditionals so not to load to ios platform. After I upgraded my Visual Studio to the latest version, the restore work properly and solved the problem.

Thank you for your time and patience.
